src/executor/spawn_blocking.rs

```diff
@@ -0,0 +1,37 @@
+use tokio::runtime::Handle;
+
+use crate::{concurrency_limit::ConcurrencyLimit, error::Error};
+
+use super::ExecuteSync;
+
+pub(crate) struct SpawnBlockingExecutor {
+    concurrency_limit: ConcurrencyLimit,
+    handle: Handle,
+}
+
+impl SpawnBlockingExecutor {
+    pub(crate) fn new(max_concurrency: Option<usize>) -> Self {
+        let concurrency_limit = ConcurrencyLimit::new(max_concurrency);
+
+        Self {
+            concurrency_limit,
+            handle: Handle::current(),
+        }
```

Comment:
I think we will want a constructor where you can pass the explicit handle you want to use

Reply:
Good point, I can add that in.